src/main/java/com/keeper/homepage/domain/post/application/convenience/CategoryFindService.java

```diff
@@ -0,0 +1,24 @@
+package com.keeper.homepage.domain.post.application.convenience;
+
+import static com.keeper.homepage.global.error.ErrorCode.POST_CATEGORY_NOT_FOUND;
+
+import com.keeper.homepage.domain.post.dao.category.CategoryRepository;
+import com.keeper.homepage.domain.post.entity.category.Category;
+import com.keeper.homepage.global.error.BusinessException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class CategoryFindService {
+
+  private final long VIRTUAL_CATEGORY_ID = 1;
```

Comment:
상수처리라면 `private static final`도 괜찮을 것 같아요

물론 스프링에서 관리해주는 싱글턴이라 `VIRTUAL_CATEGORY_ID` 변수가 여러개 생기진 않겠지만 ㅎㅎ...

```suggestion
  private static final long VIRTUAL_CATEGORY_ID = 1;
```

Reply:
감사합니다 ㅎㅎ